src/test/groovy/com/meltwater/docker/compose/DockerComposeTest.groovy

```diff
@@ -28,12 +28,12 @@ class DockerComposeTest extends Specification {
         then:
             inspectData.size() == 3
             inspectData[0].state.running
-            inspectData[0].name == "/composetest_bar-host_1"
+            inspectData[0].name == "/compose_test_bar-host_1"
```

Comment:
Instead of changing these (which breaks on my machine, there an - is inserted instead of a _)  change on line 12 to:

              compose = new DockerCompose("simple-busybox.yml", "composetest", new HashMap<String, String>())   

Then this test passed without these 3 changes.